### PR TITLE
Feature/basic app

### PR DESCRIPTION
**Pull request by iam-Akshat**
- Use API to show weather data
- Use Async/Await to handle network requests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (wathigo) [inline package.json]: **REQUIRED**
*  Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png)).
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this milestone.
Let's make a few changes before we move on.

### REQUIRED
* It uses Webpack ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/webpack_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).
* All webpack related packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).
* It changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app)).

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes requried :muscle: 
Hi @iam-Akshat :hand: 
Awesome job on this project so far. But before moving on we need to make some changes.

- [ ] The page needs to have a toggle control to determine if the result will be shown in Celsius or Fahrenheit.
Screenshot from 2021-03-28 11-25-22

- [x] Right now only the image shows the forecast (clear, rain...). It would be nice to also include a label to show it.


Happy coding :computer: 
**Sérgio Torres**

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hello again @iam-Akshat :hand: 
We still need to work on the changes requested in the last review. Please let me know if the instructions were not clear, I will help you out :smiley: 

- [ ] Just like it's saying in the instructions, the page needs to have a toggle control to determine if the result will be shown in Celsius or Fahrenheit. So if the default temperature is represented in Celsius. clicking at the toggle you can change it to Fahrenheit. This convertion can be implemented using code, there is no need to do another request in the API.
Screenshot from 2021-03-28 11-25-22

Happy coding :computer: 
**Sérgio Torres**

**Review by Forison — CHANGES_REQUESTED** [review]
#### Changes Required❌

# Hi @iam-Akshat, 

#### After cloning the branch and going through all the steps outline in the readme, below is I got on the branch
Screenshot from 2021-03-28 16-10-26
😊
Screenshot from 2021-03-28 16-10-53
I don't know if your attempt to address the earlier recommendation cause an error. Let us Address this and make sure all the earlier recommendation are addressed.😇

#### Kindly address the issue and submit a re-review🙂

#### Happy coding

Addo Forison

**Review by Forison — APPROVED** [review]
# Status::Approved✔️

# Good job, Readme points to the right direction, kindly proceed to the next task 😇.

#### Happy coding👨‍💻

Addo Forison

**Pull request merged**